manim/animation/creation.py

```diff
@@ -76,15 +121,25 @@ def construct(self):
         "lag_ratio": 1,
     }
 
-    def get_bounds(self, alpha):
+    def _get_bounds(self, alpha):
         return (0, alpha)
 
 
 class Uncreate(ShowCreation):
+    """Like :class:`ShowCreation` but in reverse.
```

Comment:
**Suggestion**: Specify that it removes the object from the scene as well than showing ShowCreation in reverse.

Reply:
I'm of two minds about this. On the one hand, `Uncreate` has an attribute called `remover` which is set to `True`. If we had better documentation, then that would be enough to document the fact that this Animation removes the mobject.

On the other hand, we of course do _not_ have good documentation and that's not currently documented anywhere. TBH I don't think this is a big deal, and it _will_ get documented at some point, as soon as we document the `Animation.remover` attribute ....